app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -37,7 +42,14 @@ def convert_to_group_trigger(miq_alert)
       when "mw_accumulated_gc_duration"
         context = { 'dataId.hm.type' => 'counter', 'dataId.hm.prefix' => 'hm_c_' }
       end
-      ::Hawkular::Alerts::Trigger.new('id'          => "MiQ-#{miq_alert[:id]}",
+
+      trigger_id = if operation == :new
+                     @ems.miq_id_prefix("alert-#{miq_alert[:id]}")
```

Comment:
So, then triggers created in Hawkular Alerting will start with MiQ-region-xyz-ems-abc-alert-id. am I right ?
I know that create an integration test from MiQ is not trivial, but all the flow could be validated ?